File: hw/otavalo/u3/hw35StrategyPersistence/Srategy/StrategyJava/java/InstrumentManagement/src/main/java/ec/edu/espe/instrument/utils/Persistence.java
package ec.edu.espe.instrument.utils;

import ec.edu.espe.instrument.model.Instrument;
import java.util.List;
/**
 *
 * @author Arelys Otavalo, the POOwer Rangers of Programming
 */

public interface Persistence {
    public boolean create(Instrument instrument);
    public List<Instrument> read(); 
    public boolean update(String id, Instrument instrument);
    public boolean delete(String id);
    public Instrument find(String id);
}

File: hw/collaguazo/u3/hw35Strategy/StrategyJs/InstrumentsManagementJS/src/main/javascript/ec/edu/espe/instruments/utils/Persistence.js
class Persistence {
  async create(instrument) { throw new Error("Not implemented"); }
  async read() { throw new Error("Not implemented"); }
  async update(id, instrument) { throw new Error("Not implemented"); }
  async delete(id) { throw new Error("Not implemented"); }
  async find(id) { throw new Error("Not implemented"); }
}

module.exports = Persistence;
